### PR TITLE
Fix tracker rotations breaking gui

### PR DESCRIPTION
**Pull request by Louka3000**
:FA_FoxetteAngy: https://github.com/SlimeVR/SlimeVR-Server/pull/307
*Quaternion lib changed and Anna forgor to change stuff*

### CODE REVIEW

**Review by TheDevMinerTV — APPROVED** [review]

File: gui/src/components/tracker/TrackersTable.tsx [hunk 1]
```diff
@@ -40,7 +40,9 @@ export function TrackerRotCol({ tracker }: { tracker: TrackerDataT }) {
   return (
     <Typography color="secondary">
       <span className="whitespace-nowrap">
-        {`${rot.x.toFixed(0)} / ${rot.y.toFixed(0)} / ${rot.z.toFixed(0)}`}
+        {`${rot.pitch.toFixed(0)} / ${rot.yaww.toFixed(0)} / ${rot.roll.toFixed(
```
Comment (Eirenliel) [inline gui/src/components/tracker/TrackersTable.tsx:43]: is it suppoed to be `yaww` not `yaw`?

Comment (Louka3000) [reply gui/src/components/tracker/TrackersTable.tsx:43]: ooops

Comment (Louka3000) [reply gui/src/components/tracker/TrackersTable.tsx]: fixed

**Review by loucass003 — APPROVED** [review]

**Review by ButterscotchV — APPROVED** [review]

**Review by ButterscotchV — CHANGES_REQUESTED** [review]
This reports rotation in radians, not degrees, please convert to degrees for the display :3

**Review by ButterscotchV — APPROVED** [review]

**Pull request merged**